rotkehlchen/chain/ethereum/modules/airdrops/decoder.py

```diff
@@ -251,6 +267,7 @@ def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
             CONVEX: (self._decode_fpis_claim, 'convex'),
             FOX_DISTRIBUTOR: (self._decode_fox_claim,),
             ELFI_LOCKING: (self._decode_elfi_claim,),
+            self.vcow.evm_address: (self._decode_cow_claim,),
```

Comment:
no need to do this. Can't you just move this to the `CowswapAirdropDecoder` and add it to this by calling super().addresses_to_decoders()?